### PR TITLE
update getting started

### LINKED ISSUE #1040: Getting started for React is not updated
The getting started React topic is still based in `create-react-app`. It needs an update with `18.3.0`.

https://www.infragistics.com/products/ignite-ui-react/react/components/general-getting-started

### PR DESCRIPTION
**Pull request by dkamburov**
closes #1040

### CODE REVIEW

File: doc/en/components/general-getting-started.md [hunk 1]
```diff
@@ -288,19 +316,20 @@ npm run build
...
 
 ```cmd
+npm install --save {PackageCommon}
 npm install --save {PackageCharts} {PackageCore}
 npm install --save {PackageExcel} {PackageCore}
 npm install --save {PackageGauges} {PackageCore}
 npm install --save {PackageGrids} {PackageCore}
 npm install --save {PackageMaps} {PackageCore}
 npm install --save {PackageSpreadsheet} {PackageCore}
+npm install --save {PackageComponents}
```
Comment (HUSSAR-mtrela) [inline doc/en/components/general-getting-started.md:332]: the "PackageComponents" variable does not exist in docsConfig. Did you forget to add it or did you mean to use "PackageCommon" which is already listed above?

File: doc/en/components/general-getting-started.md [hunk 2]
```diff
@@ -355,15 +388,13 @@ We are now ready to use the {ProductName} map component in our markup! Let's go
...
-            zoomable="true" />
-        </div>
-    );
+function App() {
+  return (
+    <div style={{ height: "100%", width: "100%" }}>
+      <IgrGrid id="grid1" data={localData} autoGenerate="true" />
+      <IgrGeographicMap width="800px" height="500px" zoomable="true" />
+    </div>
+  );
```
Comment (HUSSAR-mtrela) [inline doc/en/components/general-getting-started.md:397]: why are we adding a IG grid without instruction how to define actual data? Getting started needs to be as simple as possible so please keep just GeoMap.

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/general-getting-started.md (hunk 1 above)
Comment (dkamburov) [reply doc/en/components/general-getting-started.md:332]: Removing it

File: doc/en/components/general-getting-started.md (hunk 2 above)
Comment (dkamburov) [reply doc/en/components/general-getting-started.md:393]: All comments should be resolved now

**Pull request merged**